docs/operation/cluster_and_service_configuration.rst

```diff
@@ -0,0 +1,243 @@
+and can be customized at any point in time given that Admins follow the
+documented procedure to the later.
+
+Managing Authentication
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Add a local static user
+"""""""""""""""""""""""
+
+Local authentication via static users is enabled by default after a fresh
+MetalK8s installation.
+
+   .. important::
+
+      To continue using MetalK8s in cases where the external authentication
+      system fails, we advise MetalK8s administrators to leave the default
+      super admin account enabled at all times.
+
+To add a new static user, perform the following operations:
+
+#. Generate a password hash. The command below requires **apache-utils**
+   package to be installed.
+
+   .. code-block:: shell
+
+      $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':\n'
+
+#. From the Bootstrap node, edit the ConfigMap ``metalk8s-dex-config`` and then
+   add a new entry to the already existing list using:
+
+   .. code-block:: shell
+
+      $ kubectl --kubeconfig /etc/kubernetes/admin.conf \
+         edit configmaps metalk8s-dex-config -n metalk8s-auth
+
+   The new entry should be unique and possess mandatory fields like ``email``,
+   ``hash``, ``username`` and ``userID`` like in the example below.
+
+   .. code-block:: yaml
+
+      data:
+         config.yaml: |-
+            spec:
+               localuserstore:
+                  userlist:
+                    - email: "<email>"
+                      hash: "<replace-with-hash>"
+                      username: "<username>"
+                      userID: "<uuidv4>"
+
+
+   To obtain a unique ``UserID``, run the following command and replace the
+   <uuidv4> field above with its output.
+
+   .. code-block:: shell
+
+      python -c 'import uuid; print uuid.uuid4()'
+
+#. Save and apply the changes.
+
+
+#. Connect to the Salt master container using:
+
+   .. code-block:: shell
+
+      $ crictl exec -it $(crictl ps -q --label io.kubernetes.container.name=salt-master) bash
+
+#. From the Salt master container, apply the following salt-states to propagate
+   the new changes down to the underlying services.
+
+   .. code-block:: shell
+
+      $ salt-run state.sls metalk8s.addons.dex.deployed saltenv=metalk8s-<version>
+
+#. Finally, create and apply the required :file:`ClusterRoleBinding.yaml` file
+   that ensures that the newly added static user is bound to a Cluster Role.
+
+   .. note::
+
+      MetalK8s installations come with already existing Cluster Roles.
+      Administrators can create new Cluster Roles or refer to the existing
+      Cluster Roles.
+
+      To obtain the list of available Cluster Roles in a MetalK8s cluster,
+      use the following command:
+
+      .. code-block:: shell
+
+         kubectl --kubeconfig /etc/kubernetes/admin.conf get clusterroles
+
+      For more information about a Cluster Role, run the following command to
+      describe it.
+
+      .. code-block:: shell
+
+         kubectl describe clusterroles <name>
```

Comment:
Add the `--kubeconfig` here also please

Reply:
Also to be consistent put a `$` everywhere or no where in the code-block.

Since you are not giving any output example for me you should get rid of `$` but it's only my opinion